py/selenium/webdriver/remote/webdriver.py

```diff
@@ -1222,3 +1222,52 @@ def delete_downloadable_files(self) -> None:
             raise WebDriverException("You must enable downloads in order to work with downloadable files.")
 
         self.execute(Command.DELETE_DOWNLOADABLE_FILES)
+
+    # Federated Credential Management (FedCM)
+    def get_fedcm_title(self) -> str:
```

Comment:
if we have `get_` we should change those to properties to make this more pythonic

Reply:
We should also move these methods to be in a `fedcm` namespace so we can get rid of the `_fedcm_` in the method names. It's not a great look as an API. You can see a similar pattern with `switch_to`